lib/etl.js

```diff
@@ -95,8 +104,20 @@ function subscribe(rippled) {
 
     if (data.type === 'validationReceived') {
       data.reporter_public_key = connections[this.url].public_key;
+
+      // Store master key if validation is signed
+      // by a known valid ephemeral key
+      const master_public_key = manifests.getMasterKey(
+        data.validation_public_key);
+      if (master_public_key) {
```

Comment:
so it would be master if the validation_public_key is a known ephemeral, otherwise validation_public_key is assumed to be master, right?  I say we store master as a separate column, and then use it in the rowkey which is used for lookup.